tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -27,6 +30,10 @@
 #define BTP_SERVICE_ID_GATT	2
 #define BTP_SERVICE_ID_L2CAP	3
 #define BTP_SERVICE_ID_MESH	4
+#define BTP_SERVICE_ID_VCS	8
```

Comment:
Why the jump from 4 to 8? Defined by BTP?

Reply:
Yes, this depends on definitions in auto-pts. I can leave a comment here for better clarity.

Reply:
I think we can just define all services IDs here, even if some are not implemented by tester

Reply:
Done